src/pyhf/pdf.py

```diff
@@ -8,8 +8,8 @@
 from . import utils
 from . import events
 from .constraints import gaussian_constraint_combined, poisson_constraint_combined
-from .paramsets import reduce_paramsets_requirements
-from .paramview import ParamViewer
+from .parameters import ParamViewer
+from .parameters.utils import reduce_paramsets_requirements
```

Comment:
why not add this to `__all__`?

Reply:
i thought about this for a while. It occurred to me that this will probably be absorbed during another refactor, but I can put it under __all__.